build/rewrite.go

```diff
@@ -815,3 +834,33 @@ func hasComments(list *ListExpr) (line, suffix bool) {
 	}
 	return
 }
+
+// A wrapper for a LoadStmt's From and To slices for consistent sorting of their contents.
+// It's assumed that the following slices have the same length, the contents are sorted by
+// the `To` attribute, the items of `From` are swapped exactly the same way as the items of `To`.
+type loadArgs struct {
+	From     []*Ident
+	To       []*Ident
+	modified bool
+}
+
+func (args loadArgs) Len() int {
+	return len(args.From)
+}
+
+func (args loadArgs) Swap(i, j int) {
+	args.From[i], args.From[j] = args.From[j], args.From[i]
+	args.To[i], args.To[j] = args.To[j], args.To[i]
+	args.modified = true
+}
+
+func (args loadArgs) Less(i, j int) bool {
+	// Arguments with equal "from" and "to" parts are prioritized
+	equal_i := args.From[i].Name == args.To[i].Name
+	equal_j := args.From[j].Name == args.To[j].Name
+	if equal_i != equal_j {
+		// if equal_i == true and equal_j == false, return true, otherwise return false
```

Comment:
this comment doesn't seem to match the logic

Reply:
It does: in case the variables are different the return value is always equal to the value of `equal_i`. The following code would be equivalent:

    if equal_i != equal_j {
        if equal_i == true {  // implies that equal_j == false
            return true
        } else {  // equal_i == false and equal_j == true
            return false
        }
    }

Perhaps I should improve the comment to make it clearer.

Reply:
it was the '== true' that confused me.  I think just write in code-style (at least for my eyes)

// if equal_i and !equal_j, return true, otherwise false

Reply:
Fixed the comment.